rm/rm_foms.c

```diff
@@ -692,8 +695,12 @@ static int debtor_subscription_check(struct m0_fom *fom)
 	if (M0_IN(rfom->rf_sbscr.rhs_sm.sm_state,
 		  (RM_HA_SBSCR_FINAL, RM_HA_SBSCR_FAILURE))) {
 		if (rfom->rf_sbscr.rhs_sm.sm_rc != 0) {
-			M0_LOG(M0_DEBUG, "Can't subscribe to debtor with ep %s",
-			   m0_rm_remote_find(&rfom->rf_in)->rem_tracker.rht_ep);
+			struct m0_rm_remote *rem;
+			rem = m0_rm_remote_find(&rfom->rf_in);
```

Comment:
The `rem` might be NULL?
This should be ASSERTed or handled.

Reply:
Agree, let me put an ASSERT here.